notebooks/week3/02_model_serving.py

```
@@ -0,0 +1,190 @@
+# Databricks notebook source
+# MAGIC %md
+# MAGIC # Setup
+
+# COMMAND ----------
+
+# MAGIC %pip install ../power_consumption-0.0.1-py3-none-any.whl
+
+# COMMAND ----------
+
+dbutils.library.restartPython()
```

Ruff

Undefined name `dbutils`

(F821)